docs/diagrams/FindCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+participant ":Main" as Main
+participant ":FindCommand" as FC
+participant ":EventList" as EL
+participant "event :Event" as E
+
+Main -> FC: execute()
+activate FC
+FC -> EL: getEventByName(eventName)
+activate EL
+EL --> FC: return event
+deactivate EL
+
+opt Event found
+    FC -> E: findParticipants(personName)
+    activate E
+    E --> FC: return participantList
```

Comment:
I think you could update the return arrow labels to "[variable name]:[variable type]", as per the course convention? (e.g. in this case, it would be "participantList:ArrayList\<Participant\>")
![image](https://github.com/user-attachments/assets/d12d3e5d-c122-4507-9888-30e184459295)